### PR TITLE
Adds a rx option to force disable tlm

### PR DESCRIPTION
**Pull request by schugabe**
It possible to run multiple receivers at the same time to increase the number of server outputs but only one rx can use telemetry.
This PR adds a web option on the rx to force disable tlm so it's possible to use tlm with one rx.

### CODE REVIEW

File: src/html/index.html
Comment (schugabe) [reply src/html/index.html]: :sweat_smile:

**Comment by JyeSmith** [comment]
Something similar but for a future PR...

Ardu supports multiple receivers and ping-pongs between them after a dropped packet.  If there is a smart way to determine which rx is not active then it can stop sending tlm.

https://ardupilot.org/copter/docs/common-multiple-rx.html

**Comment by CapnBry** [comment]
I'm happy to hear that people are considering using multiple RX at once to drive many servos and this makes sense to be able to have one that still sends telemetry. The other RXes will receive the telemetry packet though, because they will be in RX mode still since they don't switch out to TX. Have you run any long (several hours) tests to make sure there's no hidden bug we have that shows up thanks to the telemetry packet shifting the PFD phase shift, or causing an RX lockup on those that aren't sending telemetry?

It seems to me that rx_main `ProcessRFPacket()` needs to return before the `PFDloop.extEvent(` if packet type is PACKET_TYPE_TLM / removed from the case statement? The PFD will try to adjust to the other RX's timer, which is shifted off by like 400us because telemetry happens earlier than a packet would come from the TX.

And then relatively minor, I found the webui description to be a little clunky. How about
> When running multiple receivers simultaneously from the same TX (to increase the number of PWM servo outputs), there can be at most one receiver with telemetry enabled.  Enable this option to ignore the Telem Ratio setting on the TX and never send telemetry from this receiver.

Checkmark label: Force telemetry OFF on this receiver

**Comment by schugabe** [comment]
I did not do extensive testing yet - we did talk about this option some time ago and thought it would be a nice addition to still support tlm. The tlm package is now ignored for the PDF calcuation. I will do some more tests with this change.
 
I was hoping that someone would improve my clunky description text :smile:

**Comment by CapnBry** [comment]
Great! I'll give it a short test with some servos and make sure nothing weird is happening with an extra RX just running scoreboard/logging/PFD dumps at various TLM Ratios.

On a personal note, have you seen a project doing this yet or is this pre-emptive? I'd love to know how it works in practice because it has been on my mind ever since I saw FliteTest do multiple RX and I thought, well we can do that with no telemetry! This is a better solution though to keep _some_ telemetry at least.

File: src/lib/WIFI/devWIFI.cpp
Comment (CapnBry) [inline src/lib/WIFI/devWIFI.cpp:427]: This can just be `request->send(200, "text/plain", "Force telemetry updated");` without the code that follows it (no response variable, headers, send, close, none of that, just what I've got here)

**Review by CapnBry — CHANGES_REQUESTED** [review]
Tested with 3x RX 100Hz Full mode Std telemetry (with the SX1280 PR fix on top of this, since LoRa otherwise doesn't work):
* 1x PMWP in master mode (with telemetry) mapped to CH1-CH5
* 1x PWMP as a second set of servos (tlm forced off) mapped to CH6-CH10
* 1x SX1280 as a third RX (tlm forced off) with logging

All works as expected with VBAT telemetry and all. Ran for 20 minutes no issues, with LQ down to around 60 on a walkaround with the TX. The RX with the logging reported PFD offset of 1-3 across the long test and many short tests.

Some things for the documentation of this feature that need saying:
* Failsafe is not synced between receivers, one can failsafe while others remain connected.
* A single failsafed receiver will have a harder time reconnecting due to the TX sending fewer sync packets to assist.
* Commands such as "Enable Rx Wifi" do not work on recveivers with telemetry forced off.

**Comment by CapnBry** [comment]
I'm adding the 3.0 tag on this because it was in before the deadline and I'm just waiting for that code to get deleted for approval, or I can push that change if it is easier.

**Comment by pkendall64** [comment]
I am also happy to approve once Caps suggestion is done too.

File: src/lib/WIFI/devWIFI.cpp
Comment (schugabe) [reply src/lib/WIFI/devWIFI.cpp:427]: I did try to remove the code but then the frontend shows an error message since the request does not have a response

**Review by pkendall64 — APPROVED** [review]

**Review by CapnBry — APPROVED** [review]

**Comment by schugabe** [comment]
> On a personal note, have you seen a project doing this yet or is this pre-emptive? I'd love to know how it works in practice because it has been on my mind ever since I saw FliteTest do multiple RX and I thought, well we can do that with no telemetry! This is a better solution though to keep _some_ telemetry at least.

I just saw a product posting for a graupner mz-32 radio and they allow binding of four rx for the same model so it seems that other products also support that.

**Comment by CapnBry** [comment]
Dang I was hoping you had some massive project to share that was going to use this. I've been telling users they can do this with telemetry off for months now, but nice to have a way to still get that main RX telemetry back for DP/VBAT. Can also be used to control head tracking separately on a model with not enough servo outputs on the FC!

**Pull request merged**
